packages/gestalt/src/SideNavigationItem.js

```diff
@@ -0,0 +1,155 @@
+   */
+  active?: 'page' | 'section',
+  /**
+   * When supplied, will display a [Badge](https://gestalt.pinterest.systems/badge) next to the item's label. See the [Badges](https://gestalt.pinterest.systems/SideNavigation#Badge) variant to learn more.
+   */
+  badge?: {|
+    text: string,
+    type?: 'info' | 'error' | 'warning' | 'success' | 'neutral' | 'darkWash' | 'lightWash',
+  |},
+  /**
+   * When supplied, will display a counter. See the [Counter](https://gestalt.pinterest.systems/SideNavigation#Counter) variant to learn more.
+   */
+  counter?: {| number: string, accessibilityLabel: string |},
+  /**
+   * Directs users to the url when item is selected.
+   */
+  href: string,
+  /**
+   * When supplied, will display Icon. See the [Icon](https://gestalt.pinterest.systems/SideNavigation#Icon) variant to learn more.
+   */
+  icon?: $Keys<typeof icons> | {| __path: string |},
+  /**
+   *  When supplied, will display a notification dot. See the [Notification](https://gestalt.pinterest.systems/SideNavigation#Notification) variant to learn more.
+   */
+  notificationAccessibilityLabel?: string,
+  /**
+   * Callback when the user selects an item using the mouse or keyboard.
+   */
+  onSelect: ({|
+    event:
+      | SyntheticMouseEvent<HTMLDivElement>
+      | SyntheticKeyboardEvent<HTMLDivElement>
+      | SyntheticMouseEvent<HTMLAnchorElement>
+      | SyntheticKeyboardEvent<HTMLAnchorElement>,
+    item: {|
+      label: string,
+      value: string,
+    |},
+  |}) => void,
+  /**
+   * Object detailing the label and value for this item.
+   */
+  item: {| label: string, value: string |},
+|};
+
+/**
+ * Use [SideNavigation.Item](https://gestalt.pinterest.systems/sidenavigation#SideNavigation.Item) for navigation, when the SideNavigation item navigates to a new page or sectoin.
+ */
+export default function SideNavigationItem({
+  active,
+  href,
+  item,
+  badge,
+  counter,
+  icon,
+  notificationAccessibilityLabel,
+  onSelect,
+}: Props): Node {
+  const [hovered, setHovered] = useState(false);
+
+  let itemColor = active ? 'selected' : undefined;
+  let textColor = active ? 'inverse' : 'default';
+
+  if (hovered && !active) {
+    itemColor = 'secondary';
+    textColor = 'default';
+  }
+
+  return (
+    <li className={classnames(styles.liItem)}>
+      <TapArea
+        accessibilityCurrent={active === 'page' ? active : undefined}
+        href={href}
+        onMouseEnter={() => setHovered(true)}
+        onMouseLeave={() => setHovered(false)}
+        onFocus={() => setHovered(true)}
+        onBlur={() => setHovered(false)}
+        role="link"
+        rounding={2}
+        tapStyle="compress"
+        onTap={({ event }) => onSelect?.({ event, item })}
+      >
+        <Box color={itemColor} paddingX={4} height={44} maxHeight={44} rounding={2}>
```

Comment:
sounds like we're getting rid of maxHeight here

Reply:
to minHEight here